adminapi/parse.py

```diff
@@ -97,24 +97,20 @@ def parse_query(term, filter_classes=fc, hostname=None):  # NOQA: C901
         term_parts = term.split(None, 1)
         if len(term_parts) == 2:
             hostname_part, remaining_part = term_parts
-            query_args = parse_query(
-                remaining_part, filter_classes, hostname_part
-            )
+            query_args = parse_query(remaining_part, None, hostname_part)
         else:
             hostname_part = term
             query_args = {}
 
         if any(x in hostname_part for x in _trigger_re_chars):
-            regexp_class = filter_classes['regexp']
+            regexp_class = Regexp
```

Comment:
I guess without the dict lookup it does not make sense anymore to explicitly store the Regexp class.

Reply:
Fixed.